Documentation/rbd-mirroring.md

```diff
@@ -0,0 +1,247 @@
+> ```
+
+* Get site name from secondary cluster(cluster-2)
+
+```bash
+kubectl get cephblockpools.ceph.rook.io mirroredpool -nrook-ceph --context=cluster-2 -o jsonpath='{.status.mirroringInfo.site_name}'
+```
+
+> ```bash
+> 5a91d009-9e8b-46af-b311-c51aff3a7b49
+> ```
+
+* With this Decoded value, create a secret on the primary site(cluster-1), using the site name of the peer as the name.
+
+```bash
+kubectl -n rook-ceph create secret generic --context=cluster-1 5a91d009-9e8b-46af-b311-c51aff3a7b49 --from-literal=token=eyJmc2lkIjoiNGQ1YmNiNDAtNDY3YS00OWVkLThjMGEtOWVhOGJkNDY2OTE3IiwiY2xpZW50X2lkIjoicmJkLW1pcnJvci1wZWVyIiwia2V5IjoiQVFDZ3hmZGdxN013R0JBQWZzcUtCaGpZVjJUZDRxVzJYQm5kemc9PSIsIm1vbl9ob3N0IjoiW3YyOjE5Mi4xNjguMzkuMzY6MzMwMCx2MToxOTIuMTY4LjM5LjM2OjY3ODldIn0= --from-literal=pool=mirroredpool
+```
+
+> ```bash
+> secret/5a91d009-9e8b-46af-b311-c51aff3a7b49 created
+> ```
+
+* This completes the bootstrap process for cluster-1 to be peered with cluster-2
+* Repeat the process switching cluster-2 in place of cluster-1, to complete the bootstrap process across both peer clusters.
+
+For more details, refer to the official rbd mirror documentation on
+ [how to create a bootstrap peer](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#bootstrap-peers).
+
+## Configure the RBDMirror Daemon
+
+Replication is handled by the rbd-mirror daemon. The rbd-mirror daemon
+ is responsible for pulling image updates from the remote, peer cluster,
+ and applying them to image within the local cluster.
+
+Creation of the rbd-mirror daemon(s) is done through the custom resource
+ definitions (CRDs), as follows:
+
+* Create mirror.yaml, to deploy the rbd-mirror daemon
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephRBDMirror
+metadata:
+  name: my-rbd-mirror
+  namespace: openshift-storage
+spec:
+  # the number of rbd-mirror daemons to deploy
+  count: 1
+  peers:
+    secretNames:
+      # list of Kubernetes Secrets containing the peer token
+      - "5a91d009-9e8b-46af-b311-c51aff3a7b49"
+```
+
+* Create the RBD mirror daemon
+
+```bash
+kubectl create -f mirror.yaml -n rook-ceph --context=cluster-1
+```
+
+> ```bash
+> cephrbdmirror.ceph.rook.io/my-rbd-mirror created
+> ```
+
+* Validate if `rbd-mirror` daemon pod is now up
+
+```bash
+kubectl get pods -n rook-ceph --context=cluster-1
+```
+
+> ```bash
+> rook-ceph-rbd-mirror-a-6985b47c8c-dpv4k  1/1  Running  0  10s
+> ```
+
+* Verify that daemon health is OK
+
+```bash
+kubectl get cephblockpools.ceph.rook.io mirroredpool -nrook-ceph -o jsonpath='{.status.mirroringStatus.summary}'
+```
+
+> ```bash
+> {"daemon_health":"OK","health":"OK","image_health":"OK","states":{"replaying":1}}
+> ```
+
+* Repeat the above steps on the peer cluster.
+
+ For more information on how to set up Ceph RBDMirror CRD, refer to
+ [rook documentation](https://rook.io/docs/rook/master/ceph-rbd-mirror-crd.html).
```

Comment:
We need a new section that configure peers like mentioned above with `CephBlockPool` and we just keep this section for the deployment of the the RBD mirror.

Reply:
Added the section for the same.